atlp-rwanda/rca-barfoot-team3-bn
# #184637985 fix: eslint issues

### What does this PR do?
This PR fixes the issues raised by ESLint due to the file env.sample

### Description of Task to be completed?
I corrected the mistakes in `package.json` and added src/env.sample in `.eslintignore `

### How should this be manually tested?

You need to pull from the branch ch-add-eslint
and
to check any fixes run `yarn lint`
to fix the issue you run `yarn lint:fix`

### What are the relevant pivotal tracker/Trello stories?
#184637985 Configure ESLint

Screenshots (if appropriate)
